src/types/store.js

```diff
@@ -7,6 +7,11 @@ import type { Store as ReduxStore } from 'redux'; // eslint-disable-line import/
 import type { Action as ActionsRef } from './actions';
 import type { State as StateRef } from './reducers';
 
+// For some reason eslint is having trouble pulling this from our libdefs, but
+// Flow is able to use it just fine.
+// eslint-disable-next-line import/named
```

Comment:
Maybe this is what https://github.com/benmosher/eslint-plugin-import/pull/1057 fixes, and we should upgrade the eslint plugin eslint-plugin-import. We have a similar thing in `src/types/store.js` too.